pom.xml

```diff
@@ -267,6 +267,23 @@
   <build>
     <pluginManagement>
       <plugins>
+        <!-- This lets us use logback instead of simplelogger with maven -->
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-checkstyle-plugin</artifactId>
+          <dependencies>
+            <dependency>
```

Comment:
This comment explains why they are necessary.
https://github.com/jcgay/maven-color/issues/11#issuecomment-125326792